easyauth/server.py

```diff
@@ -274,11 +274,18 @@ def key_setup(self):
         # check if keys exist in KEY_PATH else create
         assert "KEY_NAME" in os.environ, f"missing KEY_NAME env variable"
         key_path = os.getenv("KEY_PATH", "")
+
+        if not os.path.exists(key_path):
+            os.makedirs(key_path)
+
         if key_path:
-            key_path = f"{key_path}/{os.environ['KEY_NAME']}.key"
+            key_path = f"{key_path}/{os.environ['KEY_NAME']}"
```

Comment:
Removing the .key postfix is probably a good idea in the long run, but is a breaking change for anyone defining the KEY_NAME without .key.